ushahidi/platform-client
# Feature/post photo

This pull request makes the following changes:
- Improves the post photo upload flow via ushahidi/platform#1288

Test these changes by:
- add an image field to a survey
- create a new post
- add an image to that post
- test the delete and replace functionality
- save post
- visit "Timeline" and "Post Detail" to check if photo exists
- edit post
- test the delete and replace functionality

Fixes ushahidi/platform#1288

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/306)



TODO:
Caption not saving
